src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPAttendee.kt

```diff
@@ -46,10 +46,18 @@ class CSIPAttendee(
   @Column
   var auditModuleName: String? = null,
 
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "CREATE_USER_ID", insertable = false, updatable = false)
+  var createdByStaffUserAccount: StaffUserAccount? = null,
```

Comment:
Is this an n+1 problem? (same Q for modify)

Reply:
hmm -possibly I copied how OffenderAlert does it - but yes, it could possibly be an issue. Not sure whether to add an entitygraph or just check the sql when it gets run in anger and see if it's needed.